handler-egsp/src/main/java/com/expediagroup/streamplatform/streamregistry/handler/EgspKafkaStreamHandler.java

```diff
@@ -15,31 +15,71 @@
  */
 package com.expediagroup.streamplatform.streamregistry.handler;
 
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
 import org.springframework.stereotype.Component;
 
 import com.expediagroup.streamplatform.streamregistry.model.Specification;
 import com.expediagroup.streamplatform.streamregistry.model.Stream;
 
 @Component
-public class EgspKafkaStreamHandler<E extends Stream> implements Handler<E> {
-
+public class EgspKafkaStreamHandler implements Handler<Stream> {
   @Override
   public String type() {
     return "egsp.kafka";
```

Comment:
Consider declaring a constant /  using one from an existing class (if there is one)

Reply:
done